azurerm/internal/services/containers/container_group_resource.go

```diff
@@ -155,7 +156,7 @@ func resourceArmContainerGroup() *schema.Resource {
 				},
 			},
 
-			"tags": tags.ForceNewSchema(),
+			"tags": tags.Schema(),
```

Comment:
we'll also need [to update the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/container_group.html.markdown#L95) to reflect this is no longer ForceNew